webvr_ar_extension.md

```diff
@@ -133,6 +133,76 @@ display.getPlanes().forEach(plane => {
 });
 ```
 
+### Anchors
+
+AR systems tyr to estimate with the highest accuracy possible the pose of the device
+in the real world. This estimation evolves over time as the system "learns" more about
+the real world and because of this learning process, the pose estimation may vary to
+be as accurate as it can get.
+
+An anchor is the concept of specifying to the underlying tracking system that a especific
+pose in the world space is important for the app and that if the system should track changes
+in the device pose estimation to correctly update the important pose at all times.
+
+The VRDisplay interface has been augmented with functions to handle the creation, removal,
+retrieval and even handling of anchors. Anchors have a unique identifier and the pose 
+information represented by a model matrix that should be up to date.
+
+```js
+var anchor = display.addAnchor(modelMatrix);
+...
+display.removeAnchor(anchor);
+...
+var anchors = display.getAnchors();
+...
+display.addEventListener('anchorsupdated', e => {
+  e.anchors.forEach(anchor => {
+    // update your real world object with the updated anchor.modelMatrix
+  });
+});
+```
+
+It is important to note that the modelMatrix to create the anchor is assumed to be in world
+space and that its scale is (1, 1, 1).
+
+Anchors should be used in order to get the most accurate pose for a 3D model possible as the
+AR system underneath evolves.
+
+### Markers
+
+[WebARonARCore](https://github.com/google-ar/WebARonARCore) and 
+[WebARonTango](https://github.com/google-ar/WebARonTango) have an additional feature that allows
+to detect markers. Markers are printed tags that the AR system can recognize when they are in the
+line of sight of the camera and their world scale pose can be calculated. They can be very useful
+to trigger a experience or to share the same coordinate system between different devices, among
+others.
+
+There is support for 2 types of markers: QRCodes and ARMarkers. Both allow to obtain their world
+pose but in the case of QRCodes, they can contain a string. ARMakers have a unique identifier, a
```

Comment:
I'd prefer being explicit about this data's location: "but in the case of QRCodes, their `content` property contains an encoded string"

Reply:
Fixed.